### ISSUE TITLE
Difficult to reproduce results

### ISSUE THREAD

**Issue by Zhuanglong2** [YahiDar/TransRadar#1]
Hi，
Thank you very much for such an interesting piece of work. But after I re-trained your model, it's hard to reach the results in your paper, could you please tell me if you used some tricks in the training?

**Comment by YahiDar** [comment]
Radar-based models suffer from high variance, even with seed initialization some times. 
It is helpful to monitor the training throughout the epochs. You can track the validation accuracy and decide on the best epoch from there, as the model could very easily overfit in this context.

**Comment by Zhuanglong2** [comment]
> Radar-based models suffer from high variance, even with seed initialization some times. It is helpful to monitor the training throughout the epochs. You can track the validation accuracy and decide on the best epoch from there, as the model could very easily overfit in this context.

Thank you very much for your reply, my problem has been solved!
